guides/development-workflow.md

```diff
@@ -190,6 +190,7 @@ To deploy:
   ```
   git fetch
   git checkout master
+  git pull
```

Comment:
@tahb Ok, do you agree that the current instructions need to be changed though? Checking out local master without pulling, and merging into that will always give the wrong result unless your local master matches remote master. For me that means that the current instruction is worse than no instruction at all!